tests/console/yast2_ntpclient.pm

```diff
@@ -69,72 +97,100 @@ sub run {
 
     # check page new sychronization
     assert_screen 'yast2_ntp-client_new_synchronization';
+
     # select type of synchronization: server, then go next
-    send_key_until_needlematch 'yast2_ntp-client_sync_server', 'alt-s';
-    send_key 'alt-n';
-
-    # check NTP Server is displayed for changes
-    assert_screen 'yast2_ntp-client_ntp_server';
-
-    # select public ntp server
-    send_key 'alt-s';
-    assert_screen 'yast2_ntp-client_ntp_server_public';
-    wait_screen_change { send_key 'down'; };
-    send_key 'ret';
-
-    # check public ntp server is showing up and select UK
-    assert_screen 'yast2_ntp-client_public_ntp_server_opened';
-    send_key 'alt-u';
-    assert_screen 'yast2_ntp-client_public_ntp_country';
-    send_key_until_needlematch 'yast2_ntp-client_country_uk', 'up';
-    wait_screen_change { send_key 'ret'; };
-    send_key 'alt-s';
-    assert_screen 'yast2_ntp-client_public_ntp_server_uk';
-    send_key 'ret';
+    if ($sle_or_leap_at_least_15) {
+        # we can't select public server (yet!), so we manually enter it
```

Comment:
I guess we should add soft failure with bug reference here, not to forget to change behavior when public servers will be available

Reply:
I will check if a bug is already open for this, and I will add the corresponding reference in a soft failure.

Reply:
Find an open bug, I added the soft failure.